src/components/export-modal/export-graph.js

```diff
@@ -50,7 +50,7 @@ const exportGraph = ({ format, theme, graphSize, mockFn }) => {
   clone.prepend(style);
 
   // Download SVG/PNG
-  download(clone, 'kedro-pipeline', { css: 'internal' });
+  download(clone, 'kedro-pipeline');
```

Comment:
perhaps worth a comment to mention about adding `{css: 'internal' }` back in after your other PR is merged?

Reply:
I thought about it. Might as well